src/opendr/control/multi_object_search/multi_object_search_learner.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gym
+import os
+import shutil
+import igibson
+from pathlib import Path
+from stable_baselines3.common.utils import configure_logger
+from stable_baselines3.common.vec_env import VecEnv
+from opendr.control.multi_object_search.algorithm.SB3.ppo import PPO_AUX
+from opendr.control.multi_object_search.algorithm.SB3.encoder import EgocentricEncoders
+from igibson.utils.utils import parse_config
+from urllib.request import urlretrieve
+from opendr.control.multi_object_search.algorithm.evaluation import evaluation_rollout
+from opendr.control.multi_object_search.algorithm.SB3.save_model_callback import SaveModel
+from opendr.engine.constants import OPENDR_SERVER_URL
+from opendr.engine.learners import LearnerRL
+from igibson.utils.assets_utils import download_assets, download_demo_data
+
+
+class MultiObjectSearchRLLeaner(LearnerRL):
```

Comment:
This typo is found all across the files. I added suggestions for the main readme in the docs. Could you please look for it in the code and fix it?
```suggestion
class MultiObjectSearchRLLearner(LearnerRL):
```